_playwright-tests/UI/PopularRepo.spec.ts

```diff
@@ -22,52 +22,28 @@ test.describe('Popular Repositories', () => {
       await expect(page.getByRole('menuitem', { name: 'Add selected repositories' })).toBeVisible();
     });
 
-    await test.step('Select EPEL 10', async () => {
-      await page
-        .getByRole('row', { name: repoName10 })
-        .getByLabel('Select row 0', { exact: true })
-        .click();
-      await page.getByTestId('add-selected-dropdown-toggle-no-snap');
-      await page.getByRole('menuitem', { name: 'Add 1 repositories without snapshotting' });
-    });
-
-    await test.step('Select EPEL 9', async () => {
-      await page
-        .getByRole('row', { name: repoName9 })
-        .getByLabel('Select row 1', { exact: true })
-        .click();
-      await page.getByTestId('add-selected-dropdown-toggle-no-snap');
-      await page.getByRole('menuitem', { name: 'Add 2 repositories without snapshotting' });
-    });
-
-    await test.step('Select EPEL 8 and add both repos', async () => {
-      await page
-        .getByRole('row', { name: repoName8 })
-        .getByLabel('Select row 2', { exact: true })
-        .click();
+    await test.step('Add the 3 popular repos without snapshotting', async () => {
+      const repos = [repoName10, repoName9, repoName8];
```

Comment:
you are defining this 4 times at different places, instead this should be defined once globally so it can be reused ♻️